### PR TITLE
Fix fields after update

### PR DESCRIPTION
**Pull request by veilleperso**
After updating a record, we properly update fields

### CODE REVIEW

**Comment by sirupsen** [comment]
Can you fix the tests? 👍

**Comment by veilleperso** [comment]
I don't really understand how the test works and also don't understand why it fails because the patch only do on update the exact same things than on create ... So I guess all update test were wrong at first, or maybe there is a more deep problem.

I currently use my patch in production without any problem, so I'm quite confident that doing on update the same fields update than on create is the proper way to go... But I think someone with more knowledge of the internal of the project and test should try to fix this... Sorry :(

**Comment by veilleperso** [comment]
I've investigated more on the stub for patch request, and patching it seems to do the trick.
BTW, when running tests, I have a  lot of warnings about URI.escape being deprecated ... But it's out of scope of this patch :-)

**Comment by sirupsen** [comment]
Hey, sorry for the delay, if you fix the conflict I'll get this in for 1.0.0.

**Comment by sirupsen** [comment]
@Meekohi @chrisfrank this sounds like a legit bug-fix as well.

**Comment by Meekohi** [comment]
I didn't totally understand the change at a high level, but I fixed the test and merge conflict. Someone take a last look before 🚢 ?

File: test/table_test.rb [hunk 1]
```diff
@@ -148,8 +148,8 @@ def test_change_value_then_save_again_should_noop
   def test_updates_fields_to_newest_values_after_update
     record = first_record
 
-    record["Name"] = "new_name"
-    stub_patch_request(record, ["Name"], return_body: record.fields.merge("Notes" => "new animal"))
+    record[:name] = "new_name"
```
Comment (sirupsen) [inline test/table_test.rb]: Shouldn't this be a string @Meekohi as we're deprecating symbol access?

Comment (Meekohi) [reply test/table_test.rb]: Yup -- still getting the hang of all these review features I must have mixed something up with one of those commits.
```suggestion
    record["Name"] = "new_name"
```

**Comment by sirupsen** [comment]
Ready to 🚢 with the test change? 🎉

**Comment by Meekohi** [comment]
Thanks @veilleperso!

**Pull request merged**